respy/interpolate.py

```diff
@@ -5,43 +5,23 @@
 import numpy as np
 
 from respy.config import MAX_LOG_FLOAT
+from respy.parallelization import combine_and_split_interpolation
 from respy.parallelization import parallelize_across_dense_dimensions
 from respy.shared import calculate_expected_value_functions
 from respy.shared import calculate_value_functions_and_flow_utilities
 
 
-@parallelize_across_dense_dimensions
-def interpolate(
-    wages,
-    nonpecs,
-    continuation_values,
-    period_draws_emax_risk,
-    interpolation_points,
-    optim_paras,
-    options,
-):
+def interpolate(state_space, period_draws_emax_risk, period, optim_paras, options):
     """Interface to switch between different interpolation routines."""
     period_expected_value_functions = _kw_94_interpolation(
```

Comment:
But so far there is only one routine? 
So this is only for future ref? So for I do not really see the point of this function?

Reply:
Some time ago someone thought about working on different interpolation techniques and I thought some kind of general interface might be desirable for this purpose. Also, we might implement sparse grids. I am pretty indifferent to whether we should delete it or not.

Reply:
Your call?

Reply:
No think it is fine makes all sense!